pkg/volume/csi/csi_mounter.go

```diff
@@ -387,3 +442,9 @@ func removeMountDir(plug *csiPlugin, mountPath string) error {
 	}
 	return nil
 }
+
+// makeVolumeHandle returns csi-<sha256(podUID,volSourceSpecName)>
+func makeVolumeHandle(podUID, volSourceSpecName string) string {
```

Comment:
I'm a little confused. This is very similar to the Attachment name, but... slightly different? could you add a comment here explaining what this is exactly and what it's used for

Reply:
ok